starter_code/elevator.js

```diff
@@ -1,19 +1,64 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
+  constructor() {
+    this.floor = 0;
+    this.MAXFLOOR = 10;
+    this.requests = [];
+    this.passengers = [];
+    this.waitingList = [];
+    this.direction = "up";
+    this.setInterval;
+
+  }
+
+  start() {
+   this.setInterval = setInterval(() => {
+      this.update();
+      this.floorUp();
```

Comment:
Tiene más sentido meter el this.floorUp() y this.floorDown() en update()! No en el setInterval!